src/plugin.json

```diff
@@ -32,5 +32,12 @@
   "dependencies": {
     "grafanaDependency": ">=9.5.2",
```

Comment:
Does the grafanaDependency need to be updated to 10.3? Or do we want to keep supporting earlier versions just without the service account and any features that depend on it?

Reply:
Hmm the service account is added in 10.2.3, but technically we’re backwards compatible in that the managed key code early exits and isn’t applicable anyway in those cases

Reply:
Tested out on an earlier version and the rest of the plugin works so let's keep it as is.